### DISCUSSION TITLE
Hide icon

### DISCUSSION THREAD

**Discussion by systracer** [AlexPerathoner/SlimHUD#105] [Ideas]
Please add ability to hide icon in menu bar.

**Comment by AlexPerathoner** [comment]
How would you quit the app / access the settings then? 🤔
By the way, you can hide menu bar icons with this app: https://www.macbartender.com

**Comment by systracer** [comment]
Bartender itself adds an extra button to the menu bar.
With the menu icon hidden, access to the settings can be obtained simply by launching the program in Launchpad or Finder.
You can exit the program through Command+Q or add the Quit button in the settings.

**Comment by AlexPerathoner** [comment]
You're right, I see what I can do.

Btw @kaydenanderson this would solve the issue of creating a menu bar icon haha

**Comment by AlexPerathoner** [comment]
Starting v1.5.0 there will be a setting to hide / show the menu bar icon. If hidden, you'll have to start SlimHUD again, in order to access the settings window

#121
